reference/configuration/framework.rst

```diff
@@ -549,6 +549,19 @@ This is the field name that you should give to the CSRF token field of your form
 
 .. _reference-framework-csrf-protection:
 
+catch_all_throwables
+~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+.. versionadded:: 6.2
+
+    Symfony 7.0 will catch all Throwables by default. Use ``framework.catch_all_throwables``
```

Comment:
The .... option was introduced in Symfony 6.2.

The info for a new default value should not be part of the versionadded directive imho 

Any idea @javiereguiluz 

Maybe adding an additional versionchanged directive containing this info?